packages/sql/src/serializer/sql-serializer.ts

```diff
@@ -180,17 +189,9 @@ export class SqlSerializer extends Serializer {
         this.deserializeRegistry.prepend(ReflectionKind.array, deserializeSqlArray);
 
         //for databases, types decorated with Reference will always only export the primary key.
-        const referenceType = referenceAnnotation.registerType({ kind: ReflectionKind.class, classType: Object, types: [] }, {});
-        this.serializeRegistry.removeDecorator(referenceType);
```

Comment:
I'm not sure if its safe to omit these lines. These were added to make sure that all registered decorator templates that operate on classes are removed first (from deepkit/type default serializer)

Reply:
@marcj I've readded the code